src/transformStateWithClones.js

```diff
@@ -7,7 +7,26 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const clone = { ...state };
+  const clonesArray = [];
+
+  for (const key in actions) {
```

Comment:
Better use `for of` cycle, this way you won't need to write `actions[key]`, only `action` for example.